api/src/main/java/org/openmrs/validator/AlertValidator.java

```diff
@@ -23,6 +27,8 @@
 @Handler(supports = { Alert.class }, order = 50)
 public class AlertValidator implements Validator {
 
+	private static final Logger log = LoggerFactory.getLogger(AlertValidator.class);
+
```

Comment:
You should rename log to logger so it doesn't register as a bug on Sonar

Reply:
I have made the changes here. Thanks